### PR TITLE
docs: Updated documentation for starting.

### PR DESCRIPTION
**Pull request by hannesrabo**
This adds documentation for starting the project.

### CODE REVIEW

**Comment by hannesrabo** [comment]
PTAL @juliuscc @PethrusG

File: README.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 In this project we create a virtual network of cassandra instances to benchmark them. The goals is to perform smart request duplications to increase chanses of meeting strict latency guarantees.
 
+Starting the topology can be done by running the following command in the root folder of the repo:
+
+```sh
+> sudo ./pttopology
```
Comment (juliuscc) [inline README.md]: Is there a reason for including the `>` character?

Comment (ShubhankerS) [reply README.md]: We could use the '$' sign..?

Comment (hannesrabo) [reply README.md]: This file has this as the convention.

**Pull request merged**